### ISSUE TITLE
dbbackup command does not work with postgres

### ISSUE THREAD

**Issue by mick88** [django-dbbackup/django-dbbackup#229]
I am getting the following error:

```
Traceback (most recent call last):
  File "./manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\django\core\management\__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\django\core\management\__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\django\core\management\base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\django\core\management\base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\utils.py", line 111, in wrapper
    func(*args, **kwargs)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\management\commands\dbbackup.py", line 61, in handle
    self._save_new_backup(database)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\management\commands\dbbackup.py", line 74, in _save_new_backup
    outputfile = self.connector.create_dump()
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\db\base.py", line 80, in create_dump
    dump = self._create_dump()
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\db\postgresql.py", line 36, in _create_dump
    stdout, stderr = self.run_command(cmd, env=self.dump_env)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\db\postgresql.py", line 20, in run_command
    return super(PgDumpConnector, self).run_command(*args, **kwargs)
  File "c:\Users\Mick\Desktop\Projects\django\transplant_buddy_cms\lib\site-packages\dbbackup\db\base.py", line 150, in run_command
    "Error running: {}\n{}".format(command, stderr.read()))
dbbackup.db.exceptions.CommandConnectorError: Error running:  pg_dump transplant --host=127.0.0.1 --user=transplant --no-password --clean
pg_dump: too many command-line arguments (first is "--host=127.0.0.1")
Try "pg_dump --help" for more information.
```

The library passes `--user` argument instead of `--username` as documented in the manpages (`pg_dump --help`)

Issue tested with postgres 9.4 and 9.6

**Comment by ZuluPro** [comment]
Fixed with you PR #228

**Comment by ZuluPro** [comment]
@mick88 It's available on 3.1.2.
